packages/neos-ui-guest-frame/src/initializePropertyDomNode.js

```diff
@@ -74,6 +74,14 @@ export default ({store, globalRegistry, nodeTypesRegistry, inlineEditorRegistry,
                         actions.Changes.persistChanges([change])
                     ),
                     onChange: value => {
+                        const node = selectors.CR.Nodes.byContextPathSelector(contextPath)(store.getState());
```

Comment:
i though hey why not use the transientValues ... but we only calculate this dirty state for the inspector ... so i guess this is the way ;) ...

a possibly 'cleaner'? in terms of not - depending - on the node - here - and doing it before `onChange`  called - alternative could be to track what the ckeditor changed currently via `change:data` and depending on this invoke the callback?

Reply:
While probably possible, I wouldn't regard it as cleaner. Tracking CKEditor state is a whole can of worms in its own right, especially since we are only interested in changes that happened since the last time the property was focused.

Checking for actual changes before doing the remote call on the other hand *always* makes sense - and renders all tracking on CKEditor level moot. I was actually surprised that we didn't do that already :sweat_smile:

Reply:
okay you got me mostly convinced ... i just dislike callback - in this case the `onChange` - being fired when they simply shouldnt.
Nothing changed if we defocus a not - dirty - ckeditor (maybe ckeditor even has a flag for this?) so thats what i was referring to being "cleaner".

But still seb approved as well so feel free to merge ;)